t2.py

```diff
@@ -0,0 +1,51 @@
+
+def weak_encrypt(data):
+    key = "static_key"  
+    encrypted = "".join(chr(ord(c) ^ ord(k)) for c, k in zip(data, key))
+    return encrypted
+
+DB_USERNAME = "admin"  
+DB_PASSWORD = "password123"  
+API_KEY = "sk_test_supersecretapikey12345"  
+
+PRIVATE_KEY = """
+-----BEGIN PRIVATE KEY-----
+MIICdgIBADANBgkqhkiG9w0BAQEFAASCAmAwggJcAgEAAoGBAMJh2yfh3Dey8c+k
+Exbadcodefakeprivatekeydata++++++++++++++++++
+-----END PRIVATE KEY-----
+"""
```

Comment:
Storing sensitive information such as database credentials, API keys, and private keys within the source code is a severe security risk. This practice exposes critical information to anyone who has access to the source code, increasing the risk of data breaches. It's recommended to use environment variables or secure vaults for storing sensitive information.

**Label**: security

**Code suggestion**

```python
import os

DB_USERNAME = os.getenv('DB_USERNAME')
DB_PASSWORD = os.getenv('DB_PASSWORD')
API_KEY = os.getenv('API_KEY')
PRIVATE_KEY = os.getenv('PRIVATE_KEY')
```